mirdata/acousticbrainz_genre.py

```diff
@@ -0,0 +1,3800 @@
+                    0.192360565066,
+                    0.126187071204,
+                    0.031432043761,
+                    0.00537649122998,
+                    0.0285477209836
+                 ],
+                 'max':[
+                    0.944636106491,
+                    0.987443983555,
+                    0.971004068851,
+                    0.404173195362,
+                    0.0544534698129,
+                    0.777507662773
+                 ],
+                 'var':[
+                    0.0980709269643,
+                    0.072200588882,
+                    0.0500059388578,
+                    0.00271857017651,
+                    6.26871405984e-05,
+                    0.00670601474121
+                 ],
+                 'mean':[
+                    0.450265616179,
+                    0.34460362792,
+                    0.196296930313,
+                    0.0317945256829,
+                    0.00625067576766,
+                    0.0202882513404
+                 ]
+              }
+           },
+           ```
+        """
+        return load_extractor(self.path)["metadata"]["rhythm"]
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+        return jams_utils.jams_converter(
+            metadata={'features': load_extractor(self.path)}
+        )
+
+
+def load_extractor(path):
+    """Load a AcousticBrainz Dataset json file with all the features and metadata.
+
+    Args:
+        path (str): path to features and metadata path
+
+    Returns:
+        y (np.ndarray): the mono audio signal
+        sr (float): The sample rate of the audio file
+
+    """
+    if not os.path.exists(path):
+        raise IOError("path {} does not exist".format(path))
+
+    with open(path) as json_file:
+        meta = json.load(json_file)
+    return meta
+
+
+def download(data_home=None, force_overwrite=False, cleanup=True):
+    """Download the AcousticBrainz Dataset Dataset (annotations).
+    The audio files are not provided due to copyright issues.
+
+    Args:
+        data_home (str):
+            Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets`
+        force_overwrite (bool):
+            Whether to overwrite the existing downloaded data
+        cleanup (bool):
+            Whether to delete the zip/tar file after extracting.
+
+    """
+
+    # use the default location: ~/mir_datasets/AcousticBrainz Dataset
+    if data_home is None:
+        data_home = utils.get_default_dataset_path(DATASET_DIR)
+
+    download_message = """
+        Unfortunately the audio files of the AcousticBrainz Dataset dataset are not available
+        for download. If you have the AcousticBrainz Dataset dataset, place the contents into
+        a folder called AcousticBrainz Dataset with the following structure:
+            > AcousticBrainz Dataset/
+                > annotations/
+                > audio/
```

Comment:
This dataset has no audio, so I think it would make sense to not add this item in the documentation

Reply:
it is copy paste from other dataset.. I havent developed the download method yet